packaging/nnstreamer.spec

```diff
@@ -2,11 +2,12 @@
 %define		gstpostfix	gstreamer-1.0
 %define		gstlibdir	%{_libdir}/%{gstpostfix}
 %define		nnstexampledir	/usr/lib/nnstreamer/bin
-%define		tensorflow_support	0
+%define		tensorflow_support  0
```

Comment:
Q: double space?

Reply:
It's tab, My editor treated is as double space. I changed it! Thank you.